wsrep_api.h

```diff
@@ -786,13 +790,15 @@ struct wsrep {
    *
    * @param wsrep      provider handle
    * @param ws_handle  internal provider writeset handle
+   * @param meta       transaction meta data
```

Comment:
It would be possible to use opaque pointer to infer the transaction meta data. However, this would require more bookeeping from the provider, also each remote write set applier would need to assign some object to opaque pointer. Now the concurrency control can be done based on transaction meta data only, which will simplify the provider implementation somewhat.